modules/ROOT/pages/clustering/monitoring/status-check.adoc

```diff
@@ -1,6 +1,6 @@
 :description: This section describes how to monitor a database's availability with the help of the cluster status check procedure.
 
-[role=label--new-5.24 label--enterprise-edition]
+[role=enterprise-edition label--new-5.24]
```

Comment:
I think this should be changed to the following to render:
```suggestion
:page-role: enterprise-edition aura-db-enterprise new-5.24
```

Reply:
I'm not sure about Aura Enterprise. Where do you take this info from?

Reply:
Aura users can't use this procedure, it's an Admin procedure which is only used by the Aura operator.